contribute/style-guides/themes.md

```diff
@@ -2,9 +2,16 @@
 
 ## Overview
 
-**Themes are implemented in Typescript.** That's because our goal is to share variables between Grafana Typescript code and SASS files. Theme definitions are located in `packages/grafana-ui/src/themes/[default|dark|light].ts` files. `default.ts` file holds common variables like typography and spacing definitions, while `[light|dark].ts` primarily specify colors used in themes.
+**Themes are implemented in Typescript.** That's because our goal is to share variables between Grafana TypeScript and [Sass](https://sass-lang.com/) code. Theme definitions are located in the following files:
+
+- [packages/grafana-ui/src/themes/dark.ts](../../packages/grafana-ui/src/themes/dark.ts)
+- [packages/grafana-ui/src/themes/default.ts](../../packages/grafana-ui/src/themes/default.ts)
+- [packages/grafana-ui/src/themes/light.ts](../../packages/grafana-ui/src/themes/light.ts)
+
+The `default.ts` file holds common variables like typography and spacing definitions, while `[light|dark].ts` primarily specify colors used in themes.
 
 ## Usage
+
```

Comment:
Please avoid stacked headings. Add an intro, even just one sentence, to provide context to what this section is about.

Reply:
I just added a newline here to separate the headings, but I agree with you. I'm not familiar enough with this documentation to know what should go here. Suggestions?

Reply:
When I'm not sure, I usually put something like: This section provides usage guidelines.

At least it breaks up the headings, and hopefully a SME will add more later.

Reply:
Done: https://github.com/grafana/grafana/pull/22381/commits/cfecee995cf2a6aecce1d0b6178ee14204315bb7